docs/_documentations/troubleshooting.md

```diff
@@ -306,7 +242,96 @@ If you have the Codewind Language Server for Node.js Profiling extension enabled
+
+Even without errors, the **Stopped** state can occur if:
+  * The [health check endpoint](project-settings.html#health-check-endpoint) of the application is unreachable. Ensure it is set to an endpoint that is reachable.
+  * The [internal application port](#internal-application-port) is not correct.
+  * Your application uses HTTPS but the [HTTPS setting](project-settings.html#https-application) is not set.
+  * The time required for your application to start is longer than the [Project status ping timeout](#project-status-ping-timeout). Increase the timeout to a more sufficient time.
+
+<!--
+Action/Topic: Checking the application and build statuses
+Issue type: bug
+Issue link: https://github.ibm.com/dev-ex/iterative-dev/issues/459
+18.10: Still an issue
+-->
+## Troubleshooting general build status problems
+If your build fails, try the following:
+  * Inspect the build logs. The build logs can be opened from the project's context menu. In the build logs, you can view error messages that describe the type and cause of failure.
+  * If the build failure is related to the image build then view the docker build log and correct any errors in the project's Dockerfile.
+
+<!--
+Action/Topic: Creating a project and/or Checking the application and build statuses
+Issue type: bug/info
+Codewind version: 0.5.0
+OS: Windows
+IDE extension version: 0.5.0
+IDE version: Eclipse 2019-09
+Issue link: https://github.com/eclipse/codewind/issues/715
+-->
+## Projects stuck in starting or stopped state
+You might occasionally see projects stuck in the `Starting` or `Stopped` state even though the container logs say the projects are up and running. This can happen when you create a number of projects, for example, using the default and Appsody templates with Codewind 0.5.0. 
+
+**Workaround** Manually rebuild the projects that are stuck in `Starting` or `Stopped` state. To do this: 
+1. In the **Codewind Explorer** view, right-click your project and select **Build**.
+2. Wait for the project state to return to **Running** or **Debugging** in the **Codewind Explorer** view.
+
+<!--
+Action/Topic: Checking the application status 
+Issue type: bug/info
+Codewind version: 0.9.0
+Issue link: https://github.com/eclipse/codewind/issues/1269
+-->
+## Application stuck in Starting state 
+Some project templates come with no server configured by default, like Appsody Node.js. The application status cannot be determined for these types of projects because Codewind relies on application endpoints for status. Codewind determines the port of the application by inspecting the project's container information. The container may have a port exposed but since no server is available to ping on that port, the status check times out and the state is stuck on `Starting`. 
```

Comment:
...the status check times out, and the state is stuck on `Starting`.

Reply:
Fixed